venv.go

```diff
@@ -155,6 +155,22 @@ type VenvCommandRunOutput struct {
 	Exitcode int
 }
 
+func streamOutput(stream io.ReadCloser, outString *string) error {
+	var buff bytes.Buffer
+	reader := io.TeeReader(stream, &buff)
+
+	scanner := bufio.NewScanner(reader)
```

Comment:
nit: We can just get ride of the variable `reader` as it doesn't add any values in terms of readability or functionality. 

`scanner := bufio.NewScanner(io.TeeReader(stream, &buff))`

Reply:
Updated!